src/main.rs

```diff
@@ -594,19 +605,10 @@ fn run() -> Result<(), failure::Error> {
             }
             ("put", Some(put_key_matches)) => {
                 let key = put_key_matches.value_of("key").unwrap();
-                let is_file = match put_key_matches.occurrences_of("path") {
-                    1 => true,
-                    _ => false,
-                };
 
                 // If is_file is true, overwrite value to be the contents of the given
                 // filename in the 'value' arg.
-                let value_arg = put_key_matches.value_of("value").unwrap();
-                let value = if is_file {
```

Comment:
This logic duplicates that already in `commands::kv::key::put`; I have removed it here and tested that individual file upload still works as expected.

Reply:
i think this is the result of a merge problem, because i think i removed is_file from the put command to pull it up to main, but another commit put it back.